lib/schema.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+module.exports = {
+  projects: {
+    url: '/accounts/{accountId}/projects',
+    requiredFields: ['accountId'],
+    list: {
+      type: 'GET'
+    },
+    create: {
+      type: 'POST',
+      requiredFields: ['name', 'sitePrimary']
+    },
+    read: {
+      url: '/{projectId}',
```

Comment:
By the same token, this is also not a `url`. Maybe `path` makes the most sense.

However in traditional REST terms, this is not technically needed at this level. As a standard all GET, PUT, PATCH and DELETE operations expect an id as the last part of the path, so you can probably omit it here and default to this structure in the code. You should only have to include an explicit path for these operations if they deviate from standard REST format in some cases.